frontend/packages/kubevirt-plugin/src/components/vm-snapshots/snapshot-row.tsx

```diff
@@ -66,12 +56,14 @@ const getActions = (
     return [];
   }
 
-  const actions = [menuActionRestore, menuActionDelete];
+  const actions = [menuActionDelete];
```

Comment:
what about the snapshot detail later?

Reply:
snapshot detail?
the restore is now a dedicated button for each row, not a kebab option